### PR TITLE
Fix #489: Update usage doc to Document OpenShift configurations

### LINKED ISSUE #489: Update usage doc to Document OpenShift configurations as only works if we mention all three options
Hi,

Due to this [issue](https://github.com/projectatomic/vagrant-service-manager/issues/366), update existing existence to warn users that it will only work if all three options are provided.

### PR DESCRIPTION
**Pull request by brgnepal**
Fix #489

### CODE REVIEW

**Comment by praveenkumar** [comment]
LGTM, lets discuss with @LalatenduMohanty and @hferentschik

File: docs/using.rst [hunk 1]
```diff
@@ -127,6 +127,10 @@ To use ADB with Host-Based tools:
    2. ``config.servicemanager.openshift_image_name = "openshift/origin"`` - Specifies the image to be used.
    3. ``config.servicemanager.openshift_image_tag = "v1.2.0"`` - Specifies the version of the image to be used.
 
+   **Note:** Due to `this issue <https://github.com/projectatomic/vagrant-service-manager/issues/366/>`_ in
+   vagrant-service-manager, one need to use all the
+   three variables for any of the customization.
+
 4. Start the ADB using ``vagrant up``. For details consult the `Installation documentation`_.
```
Comment (Preeticp) [inline docs/using.rst:136]: Pending resolution of `Issue #366<https://github.com/projectatomic/vagrant-service-manager/issues/366/>`_ in the vagrant-service-manager plugin, one needs to use all the three variables for any customization.

File: docs/using.rst [hunk 2]
```diff
@@ -127,6 +127,10 @@ To use ADB with Host-Based tools:
    2. ``config.servicemanager.openshift_image_name = "openshift/origin"`` - Specifies the image to be used.
    3. ``config.servicemanager.openshift_image_tag = "v1.2.0"`` - Specifies the version of the image to be used.
 
+   **Note:** Due to `this issue <https://github.com/projectatomic/vagrant-service-manager/issues/366/>`_ in
```
Comment (brgnepal) [inline docs/using.rst]: Review comment here.

**Comment by praveenkumar** [comment]
@budhrg This is now not required because we already have fix which is going to part of our all new release.

**Pull request closed without merging**